tests/security/cc/netfilter.pm

```diff
@@ -0,0 +1,147 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Run 'netfilter' test case of 'audit-test' test suite
+# Maintainer: Liu Xiaojing <xiaojing.liu@suse.com>
+# Tags: poo#96540
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use lockapi;
+use mmapi 'wait_for_children';
+use audit_test qw(compare_run_log prepare_for_test upload_audit_test_logs);
+use scheduler 'get_test_suite_data';
+use Utils::Systemd qw(disable_and_stop_service);
+
+sub run {
+    my ($self) = shift;
+
+    select_console 'root-console';
+
+    zypper_call('in bridge-utils');
+    disable_and_stop_service($self->firewall);
```

Comment:
Xiaojing, as I know, firewalld should have been disabled in multiple machine setup phase, please double check and remove these lines

Reply:
Good point. It has been disabled in `setup_multimachine`, this line is duplicated, I will remove it.